packages/desktop/src/renderer/components/Channel/Channel.tsx

```diff
@@ -46,12 +46,6 @@ const Channel = () => {
     (currentChannelName === 'general' || currentChannelName === '') &&
     currentChannelMessagesCount === 0
 
-  let enableContextMenu = false
```

Comment:
Could you also adjust test "hides channel deletion for non-owners" to reflect that change? I think right now it forces opening the menu with `store.dispatch(navigationActions.openMenu({ menu: MenuName.Channel }))` to check that regular user can't delete channel but after your change test-user can just click on button to open the menu (as in 'deletes channel' test)

Reply:
Yea, I can look into it